### PR TITLE
Imagingcloud-574: Add allowedLabels and blockedLabels object detection parameters to SDK

### PR DESCRIPTION
**Pull request by dmitrypasyutin**
successful build: https://auto.dev.dynabic.com/job/platform/job/aspose.cloud/job/aspose.imaging/job/imaging-sdk-ruby/92/

### CODE REVIEW

File: docs/API_README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ end
 
 ## Documentation for API Endpoints
 
-All URIs are relative to *https://api.aspose.cloud/v3.0*
+All URIs are relative to *https://localhost:57972/v3.0*
```
Comment (alexermakoff) [inline docs/API_README.md]: Please, revert this url

File: docs/ImagingApi.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # AsposeImagingCloud::ImagingApi
 
-All URIs are relative to *https://api.aspose.cloud/v3.0*
+All URIs are relative to *https://localhost:57972/v3.0*
```
Comment (alexermakoff) [inline docs/ImagingApi.md]: please, revert the url

File: docs/API_README.md (hunk 1 above)
Comment (dmitrypasyutin) [reply docs/API_README.md]: fixed

File: docs/ImagingApi.md (hunk 2 above)
Comment (dmitrypasyutin) [reply docs/ImagingApi.md]: fixed

**Pull request merged**